lua/octo/commands.lua

```diff
@@ -2266,12 +2268,14 @@ function M.add_user(subject, login)
     utils.error "Cannot get issue/pr id"
   end
 
-  local function cb(user_id)
-    local query
+  ---@param user_ids string[]
+  local function cb(user_ids)
+    local query ---@type string
+    local user_ids_str = '["' .. table.concat(user_ids, '", "') .. '"]'
```

Comment:
If the gh.run get refactored to `gh.api.graph` then this formatting is done automatically. 

```lua
local gh = require "octo.gh"

gh.api.graphql {
  query = query, 
  f = {
    users = { "first_login", "second_login" },
  },
}
```
but needs the query to be parametrized as well. 

Up for that migration with this or could you make an issue for that